CMakeLists.txt

```diff
@@ -142,6 +142,9 @@ else()
   set (CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -Wno-format-zero-length")
   set (CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -Wno-missing-braces")
   set (CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -Wno-missing-field-initializers")
+  set (CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -Wno-unused-variable")
+  set (CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -Wno-switch")
+  set (CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -Wno-implicit-function-declaration")
```

Comment:
What's the use case for adding `-Wno-implicit-function-declaration`?  Wouldn't it be particularly useful and reveal missing includes that sholld be prsent?